calour/tests/test_ratio_experiment.py

```diff
@@ -0,0 +1,57 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--,  Calour development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from unittest import main
+
+import numpy as np
+
+from calour._testing import Tests
+import calour as ca
+from calour.ratio_experiment import RatioExperiment
+
+
+class ExperimentTests(Tests):
+    def setUp(self):
+        super().setUp()
+        self.test1 = ca.read_amplicon(self.rat1_biom, self.rat1_samp,
+                                      min_reads=10, normalize=None)
+
+    def test_from_exp(self):
```

Comment:
suggest a more complicated example:  contain > 1 samples in value1 or value2

Reply:
cool. added